ReferenceError: weakly-referenced object no longer exists during test teardown in vLLM plugin sampling tests

Tests in the vLLM plugin integration suite are failing during teardown with a [ReferenceError](https://github.com/tenstorrent/tt-xla/actions/runs/22923379899/job/66528834637#step:18:2433) when the cleanup code attempts to release PyTorch FX graph modules and dynamo bridge tensors.

Error Message
```
ERROR at teardown of test_non_greedy[qwen3_32b]
tests/conftest.py:488: in _release_dynamo_bridge_tensors
    if isinstance(obj, torch.fx.GraphModule):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E   ReferenceError: weakly-referenced object no longer exists
```

Root Cause
The `_release_dynamo_bridge_tensors()` function in `conftest.py` iterates through garbage collected objects and attempts to check their types using `isinstance()`. However, some PyTorch FX GraphModule objects are garbage collected before the cleanup code can process them, leaving dangling weak references.

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -485,15 +485,20 @@ def _release_dynamo_bridge_tensors():
     from torch_xla._dynamo.dynamo_bridge import GraphInputMatcher
 
     for obj in gc.get_objects():
-        if isinstance(obj, torch.fx.GraphModule):
-            if getattr(obj, "xla_args", None) is not None:
-                obj.xla_args = None
-        if isinstance(obj, GraphInputMatcher):
-            for ref in gc.get_referrers(obj):
-                if isinstance(ref, tuple):
-                    for d in gc.get_referrers(ref):
-                        if isinstance(d, dict):
-                            d.clear()
+        try:
+            if isinstance(obj, torch.fx.GraphModule):
+                if getattr(obj, "xla_args", None) is not None:
+                    obj.xla_args = None
+
+            if isinstance(obj, GraphInputMatcher):
+                for ref in gc.get_referrers(obj):
+                    if isinstance(ref, tuple):
+                        for d in gc.get_referrers(ref):
+                            if isinstance(d, dict):
+                                d.clear()
+        except ReferenceError:
+            # Skip objects that have been garbage collected
+            continue
 
 
 # TODO(@LPanosTT): We do not need to reset the seed and dynamo state for jax test. Yet this will

